src/components/Book.jsx

```diff
@@ -0,0 +1,40 @@
+const Book = () => {
+  return (
+    <li className="book">
+
+      <div className="left">
+        <div className="cover">
+          <div className="category">Action</div>
+          <div className="title">The Hunger Games</div>
+          <div className="author">Suzanne Collins</div>
+        </div>
+        <div className="buttons">
+          <button type="button">Comments</button>
+          <hr />
+          <button type="button">Remove</button>
+          <hr />
+          <button type="button">Edit</button>
+        </div>
+      </div>
```

Comment:
- [ ] Good job creating the book component 👍. You repeated yourself , what if you have 100 books ? I suggest you to pass in props book's data such as `title` , `author` , etc to fix that issue. Remember that for good practices you should not repeat yourself .

```bash
const Book = (props) => {
  const { obj } = props;
  const { title, author } = obj;
  return (
    <div className="book-container">
    ....
   </>)
```

Reply:
@iambenkis  Thank you dear reviewer for your review and also for the code example